src/applications/check-in/pages/Confirmation.jsx

```diff
@@ -21,8 +21,8 @@ const Confirmation = ({ appointment }) => {
           You’re checked in
         </h1>
         <p>
-          Please wait in the {clinicName} waiting room. We'll come get you when
-          it's time for your appointment to start.
```

Comment:
Looks like this was from the new eslint rule?

Reply:
yep

Reply:
Weird that it is automatically fixing some lines, but not others...